products/pluto/metadata.yml

```diff
@@ -1,675 +1,1705 @@
-  display_name: pfirm15_flag
-  description: ''
-  data_type: <FILL_ME_IN>!
-  example: '1'
-  values:
-  - - '1'
-    - <FILL_IN_THE_VALUE_DESCRIPTION!>
-- name: rpaddate
-  display_name: rpaddate
-  description: ''
-  data_type: text
-  example: <FILL_ME_IN>
-- name: dcasdate
-  display_name: dcasdate
-  description: ''
-  data_type: text
-  example: <FILL_ME_IN>
-- name: zoningdate
-  display_name: zoningdate
-  description: ''
-  data_type: text
-  example: <FILL_ME_IN>
-- name: landmkdate
-  display_name: landmkdate
-  description: ''
-  data_type: text
-  example: <FILL_ME_IN>
-- name: basempdate
-  display_name: basempdate
-  description: ''
-  data_type: text
-  example: <FILL_ME_IN>
-- name: masdate
-  display_name: masdate
-  description: ''
-  data_type: text
-  example: <FILL_ME_IN>
-- name: polidate
-  display_name: polidate
-  description: ''
-  data_type: text
-  example: <FILL_ME_IN>
-- name: edesigdate
-  display_name: edesigdate
-  description: ''
-  data_type: text
-  example: <FILL_ME_IN>
-- name: geom
-  display_name: geom
-  description: ''
-  data_type: text
-  example: <FILL_ME_IN>
-- name: dcpedited
-  display_name: dcpedited
-  description: ''
-  data_type: text
-  example: t
-  values:
-  - - t
-    - <FILL_IN_THE_VALUE_DESCRIPTION!>
-- name: notes
-  display_name: notes
-  description: ''
-  data_type: text
-  example: <FILL_ME_IN>
-- name: bct2020
-  display_name: bct2020
-  description: ''
-  data_type: text
-  example: '4089201'
-- name: bctcb2020
-  display_name: bctcb2020
-  description: ''
-  data_type: text
-  example: '50170071000'
+  - name: borough
+    display_name: borough
+    description: >-
+      The borough in which the tax lot is located. This field contains a two-character borough code.
+
+
+      Two portions of the city, Marble Hill and Rikers Island, are legally located in one
+      borough but are serviced by a different borough. The BOROUGH codes associated
+      with these areas are the boroughs in which they are legally located.
+      Marble Hill is serviced by the Bronx, but is legally located in Manhattan and has a
+      BOROUGH of MN. Rikers Island is serviced by Queens, but is legally located in the
+      Bronx and has a BOROUGH of BX.
+    data_type: text
+    readme_data_type: Alphanumeric - 2 characters
+    data_source: |
+      Department of City Planning - based on data from:
+      Department of Finance - Property Tax System (PTS)
+    example: QN
+    values:
+      - [BK, Brooklyn]
+      - [BX, Bronx]
+      - [MN, Manhattan]
+      - [QN, Queens]
+      - [SI, Staten Island]
+
+  - name: block
+    display_name: Tax block
+    description: |
+      The tax block in which the tax lot is located.
+      This field contains a one to five-digit tax block number.
+      Each tax block is unique within a borough (see BOROUGH).
+    data_type: integer
+    data_source: Department of Finance - Property Tax System (PTS)
+    readme_data_type: Numeric - 5 digits (99999)
```

Comment:
Q about `readme_data_type`: does this mean a value is up to 5 digits, not exactly 5 digits?

Reply:
I think it means exactly, but I'm not sure. The intent for this field is just to generate the readme - it's what we'll display as the type rather than our internal type - so it's copied straight from the PLUTO readme.

Reply:
this also makes me start to think about documenting our Python models, so that it's obvious what a field like this is for. Looks like there are some neat libraries like [autodoc-pydantic](https://autodoc-pydantic.readthedocs.io/en/stable/users/examples.html)